src/components/editor/editor.js

```diff
@@ -433,7 +433,7 @@ export default class Editor extends React.Component {
                     <div className="mb-3">
                       <label className="mr-auto">{l10n.map.editor_label_copy_recipient}</label>
                       <RecipientInput keys={this.state.publicKeys} recipients={this.state.recipientsCc}
-                        onChangeRecipient={({hasError}) => this.handleChangeRecipient({recipientsCcError: hasError})}
+                        onChangeRecipients={({recipients, hasError}) => this.handleChangeRecipients(recipients, hasError)}
```

Comment:
`handleChangeRecipients` does not set `this.state.recipientsCc`

Reply:
Fixed